.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parser": "@typescript-eslint/parser",
```

Comment:
we use usually https://standardjs.com/ for our projects.
is it possible to use this with TS? any pros and cons of this config?